index.bs

```diff
@@ -323,15 +323,23 @@ they are also defined by how content uses those features.
 <h3 id="leave-the-web-better">Leave the web better than you found it</h3>
 
 As you add new capabilities to the web platform,
-do so in a way that improves the overall platform,
-for example its security, privacy or accessibility characteristics.
+do so in a way that improves the overall platform.
 
 The existence of a defect in one part of the platform
 must not be used to excuse an addition or extension to the defect,
 which would further decrease the overall platform quality.
-Where possible, build new web capabilities that improve the overall platform quality
-by mitigating existing defects.
-Do not degrade existing capabilities without good reason.
+
+Consistency is not a good reason to propagate problems.
```

Comment:
We have to explain how to satisfy both this and https://www.w3.org/TR/design-principles/#consistency, which says consistency _is_ a good reason to propagate certain problems.

Reply:
Is this a case where we can simply forward-reference?  (I think that the text on consistency could be improved, mostly by referring to the priority of constituencies, but that's orthogonal.)

I would maybe suggestion:

```suggestion
Consistency is not often a good reason to propagate problems; see [[#consistency]].
```

Reply:
I think forward-referencing helps here, but folks are still going to be confused when the referenced section says things like "In some cases it makes sense to break consistency to improve usability, but the improvement should be very significant to justify this." I'm inclined to just undo this sentence, and let the new paragraph carry the meaning. Generally, the kinds of fixes it suggests don't actually break consistency.

Reply:
WFM

```suggestion
```